tests/baselines/reference/user/chrome-devtools-frontend.log

```diff
@@ -6895,8 +6897,18 @@ node_modules/chrome-devtools-frontend/front_end/network/NetworkConfigView.js(137
 node_modules/chrome-devtools-frontend/front_end/network/NetworkConfigView.js(138,34): error TS2339: Property 'disabled' does not exist on type 'Element'.
 node_modules/chrome-devtools-frontend/front_end/network/NetworkDataGridNode.js(121,13): error TS2339: Property 'style' does not exist on type 'Element'.
 node_modules/chrome-devtools-frontend/front_end/network/NetworkDataGridNode.js(241,33): error TS2339: Property 'request' does not exist on type 'ViewportDataGridNode<any>'.
+node_modules/chrome-devtools-frontend/front_end/network/NetworkDataGridNode.js(318,24): error TS2446: Property 'requestOrFirstKnownChildRequest' is protected and only accessible through an instance of class 'NetworkRequestNode'.
```

Comment:
this is wrong; the error is IN `Network.NetworkRequestNode`, where `requestOrFirstKnownChildRequest` should indeed be accessible.

Reply:
Oh. No, this matches the Typescript snippet:

```ts
class C {
    protected x = 1
}
class D extends C {
    m(c: C) {
        c.x
    }
}
```

I have NO idea if this is correct &mdash; none. zero! &mdash; but it behaves the same between TS and JS, so I think this is just something that  closure allows and we do not.

Reply:
`private` allows access cross-instance like that, so why wouldn't `protected`?

Reply:
it's not *exactly* the same instance -- it's C rather than D. If you use D (`m(d: D) { d.x }`) then it's fine.